### ISSUE TITLE
Governor

### ISSUE THREAD

**Issue by ElRepidente** [Tyler-IN/MnB2-Bannerlord-CommunityPatch#17]
Maybe also investigate the governor.
By checking all city values, it seems like the position is without any impact on the city besides -2 Loyalty due to "wrong culture" from the governor. 

greets

**Comment by KenRiehle** [comment]
Yeah agreed even if you have a Governor that is good at the task ie the games says you get a .25% increase to build time it doesn't seem to work and if your the governor with skill set it doesn't seem to help either. Good catch/.
